test/bun.js/os.test.js

```diff
@@ -2,144 +2,148 @@ import { it, expect } from "bun:test";
 import * as os from "node:os";
 
 it("arch", () => {
-  expect(["x64", "x86", "arm64"].some(arch => os.arch() === arch)).toBe(true);
```

Comment:
i think prettier isn't configured correctly

can you fix this so it doesn't cause lots of unintentional whitespace changes?